packages/charts/test/reattach.test.js

```diff
@@ -1,6 +1,7 @@
 import { expect } from '@vaadin/chai-plugins';
 import { fixtureSync, nextFrame, oneEvent } from '@vaadin/testing-helpers';
 import sinon from 'sinon';
+// import './chart-not-animated-styles.js';
```

Comment:
Should this be added or removed?

Reply:
I added it back, although the tests seem to pass without it, it might be safer to keep it to prevent eventual timing issues.